.github/workflows/llama_chatbot_build_and_deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: LLama Chatbot Stack - Build and Deploy
+
+on:
+  workflow_dispatch:
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+
+    env:
+      LLAMA_STACK_VERSION: 0.2.7
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: refs/heads/main
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Build environment setup
+        run: |
+          pip install -r requirements.txt
+          mkdir -p ~/.llama/providers.d/inline/safety/
+          mkdir -p ~/.llama/providers.d/remote/tool_runtime/
+          curl -o ~/.llama/providers.d/inline/safety/lightspeed_question_validity.yaml https://raw.githubusercontent.com/lightspeed-core/lightspeed-providers/refs/heads/main/resources/external_providers/inline/safety/lightspeed_question_validity.yaml
+          curl -o ~/.llama/providers.d/remote/tool_runtime/lightspeed.yaml https://raw.githubusercontent.com/lightspeed-core/lightspeed-providers/refs/heads/main/resources/external_providers/remote/tool_runtime/lightspeed.yaml
+
+      - name: Base Ansible Chatbot Stack build
+        run: |
+          export PYPI_VERSION=${{ env.LLAMA_STACK_VERSION }}
+          export LLAMA_STACK_LOGGING=server=debug;core=info
+          export UV_HTTP_TIMEOUT=120
+          llama stack build --config ansible-chatbot-build.yaml --image-type container
+          docker tag docker.io/library/ansible-chatbot-stack:0.2.7 ansible-chatbot-stack:0.2.7
```

Comment:
Should these not reference `${{ env.LLAMA_STACK_VERSION }}` instead of the concrete `0.2.7`?